sign-in.txt

```diff
@@ -10,7 +10,8 @@ TODO: Use the terminal and git commands to complete the PULL; MERGE; COMMIT; PUS
 
   NAME  : GitHub Username
   -----------------------------
-  Dr. B. : Bansenauer-Cascadia
+  Dr. B : Bansenauer-Cascadia
```

Comment:
Close, but you are missing one character that is in the [current develop branch](https://github.com/MobileApps-Cascadia/test-gitflow) (click on the sign-in.txt)